flows/modifiers/urns.go

```diff
@@ -60,9 +60,16 @@ func (m *URNsModifier) Apply(eng flows.Engine, env envs.Environment, sa flows.Se
 			log(events.NewErrorf("'%s' is not valid URN", urn))
 		} else {
 			if m.Modification == URNsAppend || m.Modification == URNsSet {
-				modified = contact.AddURN(urn, nil)
+				if len(contact.URNs()) >= flows.MaxContactURNs {
```

Comment:
I guess the error message says we allow 50 so we use `> flows.MaxContactURNs` and not `>= flows.MaxContactURNs`

Reply:
this is checking if the modifier should be allowed to add one more URN tho

Reply:
Right, I think the unit test added should not be an error as that has exactly 50 urns, we need one more urn to have that as error and use `>` comparison

Reply:
the unit test is adding 50 urns but the contact already has 1

Reply:
ahh right